src/plugins/multimedia/multimedia.js

```diff
@@ -680,19 +680,29 @@ $document.on( initializedEvent, selector, function( event ) {
 
 			// finally lets load safely
 			return Modernizr.load( {
-				load: "https://www.youtube.com/iframe_api"
+				load: "https://www.youtube.com/iframe_api",
+
+				//possible solution for multimedia and doaction conflict in corporate network
+				complete: function() {
+					setTimeout( function() {
+						if ( $this.hasClass( "youtube" ) === false ) {
+							console.warn( "There might be an issue loading the Youtube API" );
+							wb.ready( $this, componentName );
+						}
+					}, 1500 );
```

Comment:
Try to see if there is an element "iframe" initiator from the `window.performance.getEntries()`